packages/Chem/detectors.js

```diff
@@ -21,18 +19,19 @@ class ChemPackageDetectors extends DG.Package {
   detectRDSmiles(col) {
     if (col.type !== DG.TYPE.STRING)
       return null;
-    console.log('hmmm');
+    // return DG.SEMTYPE.MOLECULE;
     if (col.toList().every((el) => el && el.length > 1 &&
```

Comment:
You have just created an extra array for no good reason, keep in mind that this code should be as performant as possible